### ISSUE TITLE
Question regarding the interface

### ISSUE THREAD

**Issue by dcuenot** [eruvanos/openbrokerapi#70]
Hello,

When I have a look to the definition of the API in http://petstore.swagger.io/?url=https://raw.githubusercontent.com/openservicebrokerapi/servicebroker/v2.15/openapi.yaml, I saw that the body are not the same.

I will give a sample:
For `PUT /v2/service_instances/{instance_id}`, in the specification, the body is:
```
{
  "service_id": "string",
  "plan_id": "string",
  "context": {},
  "parameters": {}
}
```

However when I use your package, for the same API, the body is : 
```
{
	"service_id": "string",
	"plan_id": "string",
	"organization_guid": "string",
	"space_guid": "string"
}
```

Do you know where this difference is coming from?

Thanks for your help!
Damien

**Comment by eruvanos** [comment]
Hi,
regarding the spec: https://github.com/openservicebrokerapi/servicebroker/blob/v2.15/spec.md#provisioning

The attributes `organization_guid`, `space_guid `are required, the context object is optional, but should be included. 

Looks like the the swagger doesn't fit to the spec. :/

**Comment by dcuenot** [comment]
Hi,

Thanks for your quick answer.
However when I have a look in details in the SPEC, in says 2 things that seems to be inconsistent.
![image](https://user-images.githubusercontent.com/11711525/62448841-054ea500-b769-11e9-9578-e99c1b9c9a2a.png)

The attributes `organization_guid`, `space_guid` are required, but they are deprecated... Is a way to remove the required part? or to define a default value?

Thanks 🙏

**Comment by eruvanos** [comment]
I will have a look, maybe it is possible to be a little bit more open, if the context contains these informations.

**Comment by eruvanos** [comment]
Ok, I will work over the weekend on a new version. 
Is it fine for you to require these information within the context information?

**Comment by dcuenot** [comment]
Yes, it's perfect :)
Thanks for your help !

**Comment by eruvanos** [comment]
Hi short update, I finished a lot of open things to move to the API version 2.14 (2.15 follows).

After the release 4.0 I will include my local changes to support your use case in the version 4.1.

**Comment by eruvanos** [comment]
Fixed the issue with cedf87e67a54d413bf560efc5435c07989a5d0ce, thanks for waiting.
